Create_spSummariseEpisodes

```diff
@@ -0,0 +1,22 @@
+DROP PROCEDURE IF EXISTS spSummariseEpisodes;
```

Comment:
instead of dropping the sproc, you can do on line 3:
CREATE OR ALTER PROCEDURE spSummariseEpisodes

this will update the sproc if it already exists